dotnetapp-current/README.md

```diff
@@ -19,7 +19,7 @@ Follow these steps to run the sample locally:
 
 ```console
 dotnet restore
-dotnet run Hello .NET Core from Docker
```

Comment:
While args are not required, the reason they were specified was to illustrate how you can pass args to the containerized app.